tests/ignite/contrib/handlers/test_custom_events.py

```diff
@@ -8,55 +8,58 @@
 
 def test_bad_input():
 
-    with pytest.raises(ValueError):
-        CustomPeriodicEvent(n_iterations="a")
+    with pytest.warns(DeprecationWarning):
```

Comment:
Please, specify warn part of message to match, like above.

Reply:
Done